Makefile

```diff
@@ -2,16 +2,16 @@ VERSION = 1.1.0
 TAG = $(VERSION)
 PREFIX = nginx/nginx-prometheus-exporter
 
-.DEFAULT_GOAL:=help
-
-.PHONY: help
-help: Makefile ## Display this help
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m [VARIABLE=value...]\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
+.DEFAULT_GOAL:=nginx-prometheus-exporter
 
 .PHONY: nginx-prometheus-exporter
 nginx-prometheus-exporter: ## Build nginx-prometheus-exporter binary
 	CGO_ENABLED=0 go build -trimpath -ldflags "-s -w -X github.com/prometheus/common/version.Version=$(VERSION)" -o nginx-prometheus-exporter
 
+.PHONY: help
+help: Makefile ## Display this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "; printf "Usage:\n\n    make \033[36m<target>\033[0m [VARIABLE=value...]\n\nTargets:\n\n"}; {printf "    \033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
I don't think we need to move this, we just need to change the `.DEFAULT_GOAL`, right?

Reply:
Yes, you are right. I didn't know that, I just looked it up... since 2005 😂
https://git.savannah.gnu.org/cgit/make.git/commit/?h=3.81&id=9d5b5bd2f57cad88b2ea689bdce4f3d8662e73a4

I'll change it.